README.md

```diff
@@ -15,151 +15,37 @@ ILP wallet with hosted ledger and connector instances
-`CONNECTOR_ENABLE` | `false` | Run a connector instance
-`CLIENT_PUBLIC_PORT` | `80` | Client public port (if different from `CLIENT_PORT`)
-`CLIENT_TITLE` | `ILP Kit` | Browser title and logo
-
-##### Default five-bells-ledger environment variables 
-(used if the `API_LEDGER_URI` is not specified). You can read more about these variables in the [five-bells-ledger readme](https://github.com/interledgerjs/five-bells-ledger#step-3-run-it).
-
-Name | Default |
----- | ------- |
-`LEDGER_DB_URI` | `DB_URI`
-`LEDGER_ADMIN_USER` | `API_LEDGER_ADMIN_USER`
-`LEDGER_ADMIN_PASS` | `API_LEDGER_ADMIN_PASS`
-`LEDGER_HOSTNAME` | `API_HOSTNAME`
-`LEDGER_PORT` | `API_PORT + 1`
-`LEDGER_PUBLIC_PORT` | `CLIENT_PORT`
-`LEDGER_PUBLIC_PATH` | `ledger`
-`LEDGER_CURRENCY_CODE` | `USD`
-`LEDGER_CURRENCY_SYMBOL` | `$`
-
-## Advanced Mode
-ILP kit UI comes with an "advanced mode" for developers and advanced users. You can activate it with a hot-key: `option+d` on Mac or `alt+d` on Windows. 
-
-## Architecture
-ILP kit consists of a [Node.js](https://github.com/nodejs/node) (developed on v6.9.1) backend (REST API) and a client built using [React](https://github.com/facebook/react).
-
-### Backend (REST API)
-The backend is responsible for communicating with the ILP ledger, creating accounts, sending payments and keeping the payment history.
-
-#### API docs
-[http://interledger.org/ilp-kit/apidoc](http://interledger.org/ilp-kit/apidoc/)
-
-#### SPSP
-
-The wallet implements [SPSP](https://github.com/interledger/rfcs/blob/master/0009-simple-payment-setup-protocol/0009-simple-payment-setup-protocol.md) for initiating and receiving payments.
-
-##### Webfinger
-Webfinger is used to lookup account/user identifiers.
-
-Example request 
-```bash
-curl -X GET
-https://wallet.example/.well-known/webfinger?resource=acct:alice@wallet.example
+```sh
+$ git clone https://github.com/interledgerjs/ilp-kit
+$ cd ilp-kit
+$ npm install
+$ npm run build
+$ npm run configure
 ```
 
-Example response 
-```bash
-HTTP/1.1 200 OK
-{
-  "subject": "acct:alice@red.ilpdemo.org",
-  "links": [
-    {
-      "rel": "https://interledger.org/rel/ledgerUri",
-      "href": "https://red.ilpdemo.org/ledger"
-    },
-    {
-      "rel": "https://interledger.org/rel/socketIOUri",
-      "href": "https://red.ilpdemo.org/api/socket.io"
-    },
-    {
-      "rel": "https://interledger.org/rel/ledgerAccount",
-      "href": "https://red.ilpdemo.org/ledger/accounts/alice"
-    },
-    {
-      "rel": "https://interledger.org/rel/sender/payment",
-      "href": "https://red.ilpdemo.org/api/payments"
-    },
-    {
-      "rel": "https://interledger.org/rel/sender/pathfind",
-      "href": "https://red.ilpdemo.org/api/payments/findPath"
-    },
-    {
-      "rel": "https://interledger.org/rel/receiver",
-      "href": "https://red.ilpdemo.org/api/receivers/alice"
-    }
-  ]
-}
+Use the default option for every question in the `configure` command,
```

Comment:
https://github.com/interledgerjs/ilp-kit-cli/pull/24 I can switch this to use haikunator